contrib/kube-prometheus/manifests/prometheus/prometheus-k8s-rules.yaml

```diff
@@ -516,4 +516,52 @@ data:
         summary = "Kubelet is close to pod limit",
         description = "Kubelet {{$labels.instance}} is running {{$value}} pods, close to the limit of 110",
       }
-    
+
+    ALERT K8SPodWaiting
+      IF sum(sum_over_time(kube_pod_container_status_waiting[5m]) > 0 ) BY (job, exported_pod, instance, namespace)
```

Comment:
`exported_pod` is the metric's `pod` label prefixed by `exported_` as it would conflict with the exporters target `pod` label otherwise. In this case here, the exporter target should not have a `pod` label to begin with.

This is fine for this PR. We have to find a generalised approach to address this problem.